src/store/reducers/iotHubManagerReducer.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import 'rxjs';
+import { Observable } from 'rxjs';
+import { IoTHubManagerService } from 'services';
+import { toDeviceListModel } from 'services/models';
+import { createReducerScenario, createEpicScenario } from 'store/utilities';
+
+// Simulation reducer constants
+const EMPTY_DEVICES = toDeviceListModel();
+const initialState = { data: undefined };
+
+// ========================= Reducers - START
+const deviceListModelReducer = (state, action) => ({ ...state, data: action.payload });
+const iotHubManagerErrorReducer = (state, action) => ({ error: action.payload });
+const initialStateReducer = (state, action) => initialState;
+
+const updateModel = { type: 'DEVICES_UPDATE', reducer: deviceListModelReducer };
```

Comment:
The `model` terminology is a hold over from simulation (because we were saving a model). It doesn't make sense to use it here. `updateDevices` and `clearDevices` are clearer.

Reply:
OK